### DISCUSSION TITLE
Incomplete Chinese translation

### DISCUSSION THREAD

**Discussion by ColoZhao** [xournalpp/xournalpp#5917] [Q&A]
![image](https://github.com/user-attachments/assets/c8dedead-54e5-448c-876e-4c4465fa2cd6)
![image](https://github.com/user-attachments/assets/c08b0d59-c6b9-4472-a7b0-241e70a1131d)
Some places have simplified Chinese, but others do not. Why does this happen? How can I solve this problem?

**Comment by GithubAnon0000** [comment]
> Why does this happen? How can I solve this problem?

You can translate the rest. See [CONTRIBUTING](https://github.com/xournalpp/xournalpp/blob/master/CONTRIBUTING.md#contributing-translation-improvements).
